### ISSUE TITLE
database query delivers poor results when content is stored as flac + cue-file

### ISSUE THREAD

**Issue by MPconta** [MusicPlayerDaemon/MPD#1013]
### Describe the bug
The database query with the command "lsinfo" delivers different results when asking for a track stored in a single flac-file or for a track contained in a flac+cue-file.
The database query for a single flac-file looks like following:

```
lsinfo "austropop/Arik Brauer - Arik Brauer/02 Die Jause - Arik Brauer.flac"

file: austropop/Arik Brauer - Arik Brauer/02 Die Jause - Arik Brauer.flac
Last-Modified: 2017-11-19T19:32:42Z
Format: 44100:16:2
Time: 166
duration: 166.386
Title: Die Jause
Album: Arik Brauer
Artist: Arik Brauer
Comment: Bear Family Records 2012 -- BCD 17323, Germany
Genre: Austropop
Date: 1971
Track: 2
OK
```

And the query for a track contained in a flac+cue-file looks like this:

```
lsinfo "_test/Joe Bonamassa - Royal Tea/Joe Bonamassa - Royal Tea.cue/track0005"

file: _test/Joe Bonamassa - Royal Tea/Joe Bonamassa - Royal Tea.cue/track0005
Range: 1459.533-1753.426
Genre: aaa
Date: 2020
Comment: J&R Adventures 2020 -- JRA91132, US
AlbumArtist: Joe Bonamassa
Album: Royal Tea
Track: 05
Title: High Class Girl
OK
```

Evidently the results look very different.
One thing in the second report is definitely wrong - there is no AlbumArtist-tag stored for this track, but a Artist-tag with the same name.

The other thing, which i realize in the second report, is the missing of the Time-entry and the duration-entry. This results in problems with the MPD-clients. MPDroid shows strange timestamps like "-2562047788015215:-30:-8" instead of e.g. "03:43".
Gmpc just tells me, that there is no duration available for the track.
When the track is playing, the clients get the requested time-information.
Maybe the reason for this behaviour is, that when the track is playing, the information about it is asked with an other command. "currentsong" for example delivers time- and duration-information.

## Expected Behavior

The database query for tracks contained in flac-files, with the tags stored in a cue-file, should deliver correct results including time-information.

## Actual Behavior

The database query delivers wrong and incomplete results.

## Version

Music Player Daemon 0.22.3 (0.22.3)
Copyright 2003-2007 Warren Dukes <warren.dukes@gmail.com>
Copyright 2008-2018 Max Kellermann <max.kellermann@gmail.com>
This is free software; see the source for copying conditions.  There is NO
warranty; not even MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Database plugins:
 simple proxy upnp

Storage plugins:
 local udisks nfs curl

Neighbor plugins:
 upnp udisks

Decoders plugins:
 [mad] mp3 mp2
 [mpg123] mp3
 [vorbis] ogg oga
 [oggflac] ogg oga
 [flac] flac
 [opus] opus ogg oga
 [sndfile] wav aiff aif au snd paf iff svx sf voc w64 pvf xi htk caf sd2
 [audiofile] wav au aiff aif
 [dsdiff] dff
 [dsf] dsf
 [hybrid_dsd] m4a
 [faad] aac
 [mpcdec] mpc
 [wavpack] wv
 [modplug] 669 amf ams dbm dfm dsm far it med mdl mod mtm mt2 okt s3m stm ult umx xm
 [mikmod] amf dsm far gdm imf it med mod mtm s3m stm stx ult uni xm
 [sidplay] sid mus str prg P00
 [wildmidi] mid
 [fluidsynth] mid
 [adplug] amd d00 hsc laa rad raw sa2
 [ffmpeg] 16sv 3g2 3gp 4xm 8svx aa3 aac ac3 adx afc aif aifc aiff al alaw amr anim apc ape asf atrac au aud avi avm2 avs bap bfi c93 cak cin cmv cpk daud dct divx dts dv dvd dxa eac3 film flac flc fli fll flx flv g726 gsm gxf iss m1v m2v m2t m2ts m4a m4b m4v mad mj2 mjpeg mjpg mka mkv mlp mm mmf mov mp+ mp1 mp2 mp3 mp4 mpc mpeg mpg mpga mpp mpu mve mvi mxf nc nsv nut nuv oga ogm ogv ogx oma ogg omg opus psp pva qcp qt r3d ra ram rl2 rm rmvb roq rpl rvc shn smk snd sol son spx str swf tak tgi tgq tgv thp ts tsp tta xa xvid uv uv2 vb vid vob voc vp6 vmd wav webm wma wmv wsaud wsvga wv wve rtp:// rtsp:// rtsps://
 [gme] ay gbs gym hes kss nsf nsfe sap spc vgm vgz
 [pcm]

Filters:
 libsamplerate soxr

Tag plugins:
 id3tag

Output plugins:
 shout null fifo sndio pipe alsa ao openal pulse jack httpd recorder

Encoder plugins:
 null vorbis opus lame twolame wave flac shine

Archive plugins:
 [bz2] bz2
 [zzip] zip
 [iso] iso

Input plugins:
 file archive alsa tidal qobuz curl ffmpeg nfs mms cdio_paranoia

Playlist plugins:
 extm3u m3u pls xspf asx rss soundcloud flac cue embcue

Protocols:
 file:// alsa:// cdda:// ftp:// ftps:// gopher:// hls+http:// hls+https:// http:// https:// mms:// mmsh:// mmst:// mmsu:// nfs:// qobuz:// rtmp:// rtmps:// rtmpt:// rtmpts:// rtp:// rtsp:// rtsps:// scp:// sftp:// smb:// srtp:// tidal://

Other features:
 avahi dbus udisks epoll icu inotify ipv6 systemd tcp un

**Comment by MaxKellermann** [comment]
> MPDroid shows strange timestamps like "-2562047788015215:-30:-8" instead of e.g. "03:43".

That part very much sounds like a MPDroid bug - please report it to that project.

**Comment by MaxKellermann** [comment]
How to reproduce your problem? I don't have those files.

**Comment by MPconta** [comment]
I just sent you a mail...

**Comment by MaxKellermann** [comment]
I suppose this is fixed by fee282f49c6a998cf37ee4050971c20239f98056 and 1afa33c3c766af22c35b02ba58e84693243a4f3e

**Comment by MPconta** [comment]
Thanks a lot Max!
The time-problem is solved now. But there seem to be still some dissent with the "Artist" and "AlbumArtist"-tag. It seems, if there is no defined "AlbumArtist", there is an atomatic fallback from "Artist" to "AlbumArtist".

Furthermore the handling with the "Comment"-tag has changed. In my example here, it looked like that in the old version:
`Comment: J&R Adventures 2020 -- JRA91132, US`
Now it is much shorter:
`Comment: J&R`
Should i open a new ticket about that?
